.circleci/config.yml

```diff
@@ -11,7 +11,7 @@
 # If you change the `default_docker_image` version, also change the `cache_key` version
 var_1: &default_docker_image circleci/node:10.12
 var_2: &browsers_docker_image circleci/node:10.12-browsers
-var_3: &cache_key node-0.12-{{ checksum "yarn.lock" }}-{{ checksum "examples/program/yarn.lock" }}-{{ checksum "internal/npm_install/test/package/yarn.lock" }}
+var_3: &cache_key node-0.12-{{ checksum "yarn.lock" }}-{{ checksum "examples/program/yarn.lock" }}-{{ checksum "internal/npm_install/test/package/yarn.lock" }}-bust3
```

Comment:
Yes. Because the user node_modules are used and there were there were breaks during development after updating the package.json & lock files (lock files are cache keys). I needed to bust the cache a few times I as moved along.